storage.conf

```diff
@@ -23,3 +23,24 @@ size = ""
 
 # OverrideKernelCheck tells the driver to ignore kernel checks based on kernel version
 override_kernel_check = "false"
+
+# Remap-UIDs/GIDs is the mapping from UIDs/GIDs as they should appear inside of
+# a container, to UIDs/GIDs as they should appear outside of the container, and
+# the length of the range of UIDs/GIDs.  Additional mapped sets can be listed
+# and will be heeded by libraries, but there are limits to the number of
+# mappings which the kernel will allow when you later attempt to run a
+# container.
+#
+# remap-uids = 0:1668442479:65536
+# remap-gids = 0:1668442479:65536
+
+# Remap-User/Group is a name which can be used to look up one or more UID/GID
+# ranges in the /etc/subuid or /etc/subgid file.  Mappings are set up starting
+# with an in-container ID of 0 and the a host-level ID taken from the lowest
+# range that matches the specified name, and using the length of that range.
+# Additional ranges are then assigned, using the ranges which specify the
+# lowest host-level IDs first, to the lowest not-yet-mapped container-level ID,
+# until all of the entries have been used for maps.
+#
+# remap-user = "crio"
```

Comment:
Should we use the name of a project?  Is buildah/podman/crio are sharing this config?

Reply:
It'll usually be set to the name of a user account that usermod added.  We don't create any unprivileged accounts in any of the tools (and yes, they all share this one), but I wanted to have a placeholder value in there.  Do you have any suggestions for what this should be set to?

Reply:
storage?

Reply:
"storage" it is.  Changed it.